### PR TITLE
{bio}[foss/2023a] medaka v1.12.0

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — CHANGES_REQUESTED** [review]
@PetrKralCZ Please add `pyabpoa` as extra dependency, to avoid warnings like:
```
Cannot import pyabpoa, some features may not be available.
```

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/medaka/medaka-1.12.0-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+exts_list = [
+    ('mappy', _minimap_ver, {
+        'checksums': ['e53fbe9a3ea8762a64b8103f4f779c9fb16d418eaa0a731f45cebc83867a9b71'],
+    }),
+    ('wurlitzer', '3.1.1', {
+        'source_tmpl': SOURCE_PY3_WHL,
+        'checksums': ['0b2749c2cde3ef640bf314a9f94b24d929fe1ca476974719a6909dfc568c3aac'],
+    }),
+    ('h5py', '3.10.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/m/medaka/medaka-1.12.0-foss-2023a.eb:60]:
```suggestion
    # medaka 1.12.0 requires h5py ~=3.10.0
    ('h5py', '3.10.0', {
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3158.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/dde63a63db703b73e9510afd2dbb695a for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**